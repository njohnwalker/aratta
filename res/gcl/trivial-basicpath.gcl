@req(x <= y)
y := 30 + 14 - x ;
x := 17 ;
x := z - 7
@ens(z <= x & y <= x & y <= z & z <= 10)
