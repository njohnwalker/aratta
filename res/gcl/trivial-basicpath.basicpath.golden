Path 1:
  { PRE: x <= y }
  ; Substitute
      @y -> (30 + 14) - x
  ; Substitute
      @x -> 17
  ; Substitute
      @x -> z - 7
  { POST: z <= x & (y <= x & (y <= z & z <= 10)) }